samples/Chat/src/app/ChatScreen.tsx

```diff
@@ -104,7 +109,14 @@ export const ChatScreen = (props: ChatScreenProps): JSX.Element => {
             options={{
               autoFocus: 'sendBoxTextField',
               /* @conditional-compile-remove(chat-composite-participant-pane) */
-              participantPane: !hideParticipants
+              participantPane: !hideParticipants,
+              /* @conditional-compile-remove(attachment-upload) @conditional-compile-remove(attachment-download) */
+              attachmentOptions: {
+                /* @conditional-compile-remove(attachment-upload) */
+                uploadOptions: attachmentUploadOptions,
+                /* @conditional-compile-remove(attachment-download) */
```

Comment:
`attachment-upload` assume it needs everything from `attachment-download`, can you add both upload and download cc flag for places that are for places that are right now only 'download'?

This apply for other places in this PR as well